docs/SENTENCES.md

```diff
@@ -34,8 +41,21 @@ Feel free to set up this QA however makes most sense for you, but here's a [samp
 
-## Flagging problematic sentences
+## Rules for QA that applies (or not) to the different imputs and outputs
+To help you, for exemple with the 'not supervised, do your own QA' bulk submission, you _may_ find interesting to have a look at : 
+* [cleanup](https://github.com/common-voice/sentence-collector/tree/main/server/lib/cleanup) and [validation](https://github.com/common-voice/sentence-collector/tree/main/server/lib/validation) rules for sentence collector, and/or 
+* [language](https://github.com/common-voice/cv-sentence-extractor#using-language-rules) rules for extractor, or even 
+* [preprocessor](https://github.com/common-voice/CorporaCreator/tree/master/src/corporacreator/preprocessors) for CorporaCreator ([CorporaCreator](https://github.com/common-voice/CorporaCreator) is the Common Voice extractor that, linking text and recorded voice samples, make them available for [SST](https://en.wikipedia.org/wiki/Speech_recognition) engines' training).
+
+ 
+# Correcting existing data
+
+As you can see, some methods doesn't go through automated cleanup/validation/rules, and they are not unified ([...not yet](https://discourse.mozilla.org/t/sentence-collector-cleanup-before-export-vs-cleanup-on-upload/105411/15)). Thus, there is a process to remove old data that might need to be discarded.
+
+## Flagging (and removing) problematic sentences already in the Common Voice database
+
+If you notice sentences that need to be deleted 🚩, first check what the source of the sentence is. 
 
-If you notice sentences that need to be deleted, first check what the source of the sentence is. If the file that the sentence is located in is called `sentence-collector.txt`, that means it was automatically exported from Sentence Collector. In that case, please file an issue on the [Sentence Collector repo](https://github.com/Common-Voice/sentence-collector/) with a plaintext file of all of the problem sentences, with one sentence per line.
+If the file that the sentence is located in is called `sentence-collector.txt` (localised [here](https://github.com/common-voice/common-voice/blob/main/server/data/en/sentence-collector.txt) for english, or [here](https://github.com/common-voice/common-voice/blob/main/server/data/fr/sentence-collector.txt) for french, etc.), that means it was automatically exported from Sentence Collector. In that case, please file an [issue](https://github.com/common-voice/sentence-collector/issues) on the [Sentence Collector repo](https://github.com/Common-Voice/sentence-collector/) with a plaintext file of all problematic sentences, listed one sentence per line. (Note : it is useless to correct directly the common voice database if it comes for sentence collector, because it will automatically be imported, again, by the sentence collector. You have to clean up sentence collector's database first)
```

Comment:
```suggestion
Search for the source of the sentence within the [data](https://github.com/common-voice/common-voice/blob/main/server/data) folder. Folders are split up by language. If the sentence is found in `sentence-collector.txt`, that means it was automatically exported from Sentence Collector. In that case, please file an [issue](https://github.com/common-voice/sentence-collector/issues) with a plaintext file of all problematic sentences, listed one sentence per line. Note that in this case removing it from the file through a Pull Request will not help, because it will automatically be added again with the next export.
```

I've rephrased it a bit so examples shouldn't be necessary. What do you think?

Reply:
yes :) 👍